src/go/rdctl/pkg/factoryreset/delete_data.go

```diff
@@ -30,27 +30,38 @@ import (
 
 	dockerconfig "github.com/docker/docker/cli/config"
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
-	p "github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
 	"github.com/sirupsen/logrus"
 )
 
-func addAppHomeWithoutSnapshots(appHome string) []string {
-	haveSnapshots := false
-	if snapshots, err := os.ReadDir(filepath.Join(appHome, "snapshots")); err == nil {
-		haveSnapshots = len(snapshots) > 0
+func appHomeDirectories(appPaths paths.Paths) []string {
```

Comment:
This would be a good time to add a function doc comment.  It's not obvious from the function name what this returns (since `appHome` is a _singular` directory, and the name doesn't imply its children).  Not that the old name was any better…

Reply:
I couldn't come up with a good name either. I eventually decided that it is good enough, given the context that the function is in the `factoryreset` package. But I guess another comment doesn't hurt.